EnglishFilter/sections/antiadblock.txt

```diff
@@ -638,6 +638,10 @@ scrolller.com#%#(function() { try { if ("undefined" != typeof localStorage) { lo
 ! https://github.com/AdguardTeam/AdguardFilters/issues/84871
 ||ptube.live/ads/popup/popunder/ads.js$script,redirect=noopjs,important
 @@||ptube.live/ads/popup/popunder/ads.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/85065
+appsumo.com#@#.googleAds
+appsumo.com#@#.googleAd
+appsumo.com#@#.banner_ad
```

Comment:
They are checking these elements `.ad.ads.Ad.Ads.googleAd.googleAds`, so this `appsumo.com#$#.ad.ads.Ad.Ads.googleAd.googleAds { display: block !important; }` should enough and add exception with those elements for content blockers.